staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -148,8 +148,11 @@ type Config struct {
 	// RESTOptionsGetter is used to construct RESTStorage types via the generic registry.
 	RESTOptionsGetter genericregistry.RESTOptionsGetter
 
-	// If specified, requests will be allocated a random timeout between this value, and twice this value.
-	// Note that it is up to the request handlers to ignore or honor this timeout. In seconds.
+	// If specified, all requests except those which match the LongRunningFunc predicate will timeout
+	// after this duration. In seconds.
```

Comment:
Do we need `In seconds` for type time.Duration?

Reply:
Good catch! No, we don't. I'll remove.